src/utils/ajv.ts

```diff
@@ -49,6 +63,15 @@ export function validate(
           : null
       }`,
     };
+    // return { valid: false, error: ajv.errorsText() };
+    // throw new Error(ajv.errorsText());
   }
-  return { valid: true, error: undefined };
+
+  // return data;
+
+  // }
+  return { valid: true, error: null };
+  // } catch (error) {
+  // throw console.trace(error);
```

Comment:
Are errors still logging?

Reply:
Actually this code is not used anymore, I have to remove it